src/client/components/home.js

```diff
@@ -93,18 +95,26 @@ class RequestForm extends Component {
       this.setState({ submitting: true, errors: { incompleteForm: false, network: false } });
       fetch( url, fetchOpts )
         .then( checkStatus )
-        .then( () => new Promise( resolve => this.setState({ done: true }, resolve ) ) )
+        .then( response => response.json() )
+        .then( docJSON => new Promise( resolve => this.setState({ done: true, docJSON }, resolve ) ) )
         .catch( () => new Promise( resolve => this.setState({ errors: { network: true } }, resolve ) ) )
         .finally( () => new Promise( resolve => this.setState({ submitting: false }, resolve ) ) );
     }
   }
 
   render(){
-    if( this.state.done ){
+    const { done, docJSON } = this.state;
+    if( done && docJSON ){
+      const { privateUrl, citation: { doi, title, reference } } = docJSON;
+      const articleString = _.compact([ title, reference ]).join(' ');
       return h('div.home-request-form-container', [
         h('div.home-request-form-done', [
-          h('div.home-request-form-done-icon', [ h('i.material-icons', 'check') ]),
-          h('div.home-request-form-done-msg', this.props.doneMsg )
+          h( 'a.home-request-form-done-button', { href: privateUrl, target: '_blank', }, 'START BIOFACTOID' ),
```

Comment:
Just an FYI -- not necessary to make a change here, but it's useful to know:  You can alternatively set stylistic casing through the `text-transform` css property.  That way, you don't have to change the code if you want to change the casing, and all similar buttons would have the same casing enforced.